package.json

```diff
@@ -64,6 +64,7 @@
     "accept-language-parser": "1.4.1",
     "bat-publisher": "^2.0.24",
     "bat-utils": "file:./bat-utils",
+    "bee-queue": "^1.2.2",
```

Comment:
Would be good to security team's approval here too

Reply:
yes